packages/ui-components/styles/theme.css

```diff
@@ -95,7 +94,7 @@
   --font-xs-line-height: 1rem;
   --font-sm: 0.87rem;
   --font-sm-line-height: 1.25rem;
-  --font-base: 1rem;
+  --font-base: 0.9rem;
```

Comment:
Hmmm, did we not use something else before? What were we using for defining base font size?

Reply:
IDK maybe coming from codemod ?

Reply:
Can you check what was the original value?

Reply:
https://github.com/nodejs/nodejs.org/blob/523019763c69682bcbdb85c75c6c5237594212d4/packages/ui-components/tailwind.config.ts#L90-L108

Reply:
I looked to the Wayback machine and apparently it was always 16px. That's interestingly odd. Let's undo this change then and keep base as 1rem.

Reply:
It just feels odd to me that the content font size is larger than, for example, the sidebar or other elements such as tables and so on. What do you all think, @nodejs/nodejs-website?